fuzz/fuzz_targets/frame.rs

```diff
@@ -0,0 +1,17 @@
+#![cfg_attr(not(windows), no_main)]
+
+#[cfg(not(windows))]
```

Comment:
Can we simply avoid building this package on windows?  Because having this annotation everywhere is going to get tiresome.

Reply:
I had looked into this, but wasn't able to figure out how to do this.